amazon-cloudwatch/config-map.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+data:
+  # Configuration is in Json format. No matter what configure change you make,
+  # please keep the Json blob valid.
+  cwagentconfig.json: |
+    {
+      "structuredlogs": {
```

Comment:
@martina-if, any reason for `structuredlogs` here?
Going through the [Amazon docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-metrics.html#create-configmap) again, it looks like the `ConfigMap` available at https://s3.amazonaws.com/cloudwatch-agent-k8s-yamls/kubernetes-monitoring/cwagent-configmap.yaml has `logs` instead.
Also, this seems to break the setup:

#### With `structuredlogs`

```console
$ kubectl get po -n amazon-cloudwatch
NAME                       READY   STATUS             RESTARTS   AGE
cloudwatch-agent-x9qrv     0/1     CrashLoopBackOff   5          4m25s
```
```console
$ kubectl -n amazon-cloudwatch logs cloudwatch-agent-x9qrv
[...]
E! Error: no outputs found, did you provide a valid config file?
```

#### With `logs`

```console
$ kubectl get po -n amazon-cloudwatch
NAME                       READY   STATUS    RESTARTS   AGE
cloudwatch-agent-7sqp2     1/1     Running   0          4s
[...]
```

Reply:
"Fixed" in https://github.com/weaveworks/eks-gitops-example/pull/11, but still interesting in more details to ensure this is the right "fix".